example-config.yml

```diff
@@ -332,6 +332,27 @@ itinerary:
 #       - WALK
 #       - BICYCLE
 
+### Configure here the carbon output in grams per meter for each mode.
+### massUnits can be 'pound', 'kilogram', 'ounce', or 'gram'
+# co2:
+#   carbonIntensity:
+#     walk: 0
+#     bicycle: 0
```

Comment:
Seeing NaN for bicycle_rent -- we probably should have a better null check and also include a value for bike_rent 

<img width="425" alt="Screen Shot 2022-02-09 at 7 48 55 AM" src="https://user-images.githubusercontent.com/86619099/153237224-b91f1e00-e107-444a-99b0-8864634c453a.png">

Reply:
I thought I fixed that!